BLAG.Common/Models/QuestionAudio.cs

```diff
@@ -1,13 +1,11 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Text;
+﻿
+using System;
 
 namespace BLAG.Common.Models
 {
     public class QuestionAudio : QuestionBase
     {
-        [Required]
-        public byte[] Audio { get; set; }
+        // String will hold the ID of where the audio file is saved in LiteDB
+        public String Audio { get; set; }
```

Comment:
You should use the keyword `string` instead of the type name `String`. If resharper didn't warn you let me know.